pkg/purl/purl.go

```diff
@@ -211,8 +210,6 @@ func (p *PackageURL) LangType() ftypes.LangType {
 			return ftypes.GKE
 		case NamespaceAKS:
 			return ftypes.AKS
-		case NamespaceRKE:
```

Comment:
It seems like RKE2 is entirely different from RKE https://docs.rke2.io/#how-is-this-different-from-rke-or-k3s
I think we should change this to be rke2

Reply:
`NamespaceRKE` has been removed as rancher do not maintain their own fork , they use upstream